conbench/api/benchmarks.py

```diff
@@ -113,10 +113,17 @@ def get(self):
             for benchmark_result in benchmark_results:
                 benchmark_result.z_score = 0
         elif batch_id:
-            benchmark_results = BenchmarkResult.all(batch_id=batch_id)
+            batch_id_list = batch_id.split(",")
```

Comment:
`batch_id` should be renamed to `batch_id_arg` so it does not imply one batch_id.
`batch_id_list` should be renamed to `batch_ids`, which is consistent with how vars for lists are named